packages/vkui/src/components/SplitCol/SplitCol.tsx

```diff
@@ -93,6 +98,10 @@ export const SplitCol = (props: SplitColProps) => {
     animate,
   });
 
+  if (process.env.NODE_ENV === 'development' && spaced !== undefined) {
+    warn('Свойство spaced устарело и будет удалено в v6. Используйте autoSpaced', 'error');
```

Comment:
```suggestion
    warn('Свойство spaced устарело и будет удалено в v6. Используйте autoSpaced');
```
Технически это не (совсем) ошибка, так что варнинга достаточно будет. 😉